IOSExercises/Views/ProfileViewController.m

```diff
@@ -0,0 +1,104 @@
+//
+//  ProfileViewController.m
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 10/27/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "ProfileViewController.h"
+
+@interface ProfileViewController ()
+@property (nonatomic, strong) NSMutableArray *landscapeConstraints;
+@end
+
+@implementation ProfileViewController
+
+typedef NSString * SupportedRegion NS_STRING_ENUM;
```

Comment:
Declare static properties/constants at the start of file after the imports

Note that these properties have already been declared in a different file